var/spack/repos/builtin/packages/py-morphio/package.py

```diff
@@ -30,3 +30,8 @@ class PyMorphio(PythonPackage):
         depends_on("py-h5py@3", type=("build", "run"))
     else:
         depends_on("hdf5", type=("build", "run"))
+
+    depends_on("highfive", type=("build", "run"))
+    depends_on("py-pybind11", type=("build", "run"))
+    # mpi is an indirect dependency, but the build fails if we don't specify it explicitly
+    depends_on("mpi", type=("build", "run"))
```

Comment:
Silly question: but should all the MPI stuff not be a variant?

Reply:
I could make that change, but I'd argue for having it enabled by default as our build doesn't work without it anyway.